ipatests/test_webui/ui_driver.py

```diff
@@ -1615,6 +1615,34 @@ def get_t_vals(t):
             # add multiple at once and test table delete button
             self.add_table_associations(table, keys, delete=True)
 
+    def add_sshkey_to_user(self, user, ssh_key):
+        """
+        Add ssh public key to particular user
+
+        user (str): user to add the key to
+        ssh_key (str): public ssh key
+        """
+        self.navigate_to_entity('user')
+        self.navigate_to_record(user)
+
+        ssh_pub = 'div[name="ipasshpubkey"] button[name="add"]'
+        self.find(ssh_pub, By.CSS_SELECTOR).click()
+        self.wait_for_request(n=5, d=2)
```

Comment:
Why wait for 5 requests with 2-second wait at the end? Clicking on add btw AFAIK doesn't cause any request. Simple `self.wait()` could be fine here.